source-mixpanel-native/source_mixpanel_native/streams/base.py

```diff
@@ -26,13 +26,6 @@ class MixpanelStream(HttpStream, ABC):
 
     DEFAULT_REQS_PER_HOUR_LIMIT = 60
 
-    @property
```

Comment:
Just noting that this will impact all the mixpanel streams and not just the Exports one - but I assume that if anything that's probably a good thing, since other APIs may not have the sorting properties needed for it to work correctly.

Reply:
Some of the other streams (`engage`, `cohort_members`, `funnels`) are overriding the `state_checkpoint_interval` from their parent, so they shouldn't be affected by this change. 

The streams that no longer have a `state_checkpoint_interval` property are `annotations` and `cohorts`. Both of these only work as full refresh, so they shouldn't have `state_checkpoint_interval` anyway.